frontend/src/container/NewDashboard/DashboardVariablesSelection/util.ts

```diff
@@ -29,3 +29,110 @@ export const convertVariablesToDbFormat = (
 		result[id] = obj;
 		return result;
 	}, {});
+
+const getDependentVariables = (queryValue: string): string[] => {
+	const variableRegexPattern = /\{\{\s*?\.([^\s}]+)\s*?\}\}/g;
```

Comment:
we do support a bunch of other variable formats as well, please do include them as well